mod/state-transition/pkg/core/state_processor_genesis_test.go

```diff
@@ -160,53 +128,35 @@ func checkValidatorNonBartio(
 }
 
 func TestInitializeBartio(t *testing.T) {
-	// Create state processor to test
-	cs := spec.TestnetChainSpec()
-	execEngine := mocks.NewExecutionEngine[
-		*types.ExecutionPayload,
-		*types.ExecutionPayloadHeader,
-		engineprimitives.Withdrawals,
-	](t)
-	mocksSigner := &cryptomocks.BLSSigner{}
-
-	sp := createStateProcessor(
-		cs,
-		execEngine,
-		mocksSigner,
-		dummyProposerAddressVerifier,
-	)
-
-	// create test inputs
-	kvStore, err := initStore()
-	require.NoError(t, err)
-	beaconState := new(TestBeaconStateT).NewFromDB(kvStore, cs)
+	cs := setupChain(t, "testnet")
```

Comment:
nit: shouldn't we have a variable for "testnet", like `components.BetnetChainSpecType` above?

Reply:
Right now anything thats not from one of these:
```golang
	DevnetChainSpecType = "devnet"
	BetnetChainSpecType = "betnet"
	BoonetChainSpecType = "boonet"
```
will default to the `TestnetChainSpec()`. I'm personally okay with this for now. We could maybe rename this to `LocalnetChainSpec()` if you like.